### PR TITLE
Dflt autodateformat

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by tacaswell** [comment]
don't merge, this still needs to have comments from #5445 addressed.

**Comment by tacaswell** [comment]
I think I have addressed all of @mdboom and @WeatherGod 's comments in #5445.

This can either be merged as-is or folding into #5774

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 """
 
 from __future__ import print_function, absolute_import
-
+from string import Template
```
Comment (mdboom) [inline setup.py]: Weird that I've never used this, but it seems like the right tool for the job...

**Comment by mdboom** [comment]
Merged this.  I'll `git rebase -i` it out of #5774

**Pull request merged**